### ISSUE TITLE
[terra-paginator] [IE] Shift+Tab sets focus on the last item after paginating to the last page instead of the Previous button

### ISSUE THREAD

**Issue by KunigundaLauly** [cerner/terra-core#3428]
# Bug Report
In IE, the Shift+Tab sets focus on the last item (or if there is no selectable item, previous element) after paginating to the last page instead of the Previous button.

## Description

After pagination with keyboard to the last page, Shift + Tab should go to the "<Previous" button.

Expected example, in Chrome:
![chrome-paginator-tab](https://user-images.githubusercontent.com/47626274/116420683-805dc600-a83e-11eb-8ed2-1a357d07af02.gif)

Problem in IE:
![ie-paginator-tab](https://user-images.githubusercontent.com/47626274/116420707-8784d400-a83e-11eb-9378-df4f1c2c669a.gif)

## Steps to Reproduce

1. Open the https://engineering.cerner.com/terra-ui/components/terra-paginator/paginator/progressive-paginator-controlled 
page in IE.
2. Press Tab until the "Last" button is selected
3. Press Enter.
4. Press Shift + Tab

Current result:
The focus is on the "Set Page to 45" link.

Expected result:
The focus is on the "<Previous" button.

## Additional Context / Screenshots

We got this requirement for our project, to fully support the keyboard navigation in IE.

## Expected Behavior

After pagination with keyboard to the last page, Shift + Tab should go to the "<Previous" button.

## Environment

* Component Name and Version: terra-paginator/lib/ControlledProgressivePaginator 2.72.0
* Browser Name and Version: Internet Explorer 2004
* Node/npm Version: [e.g. Node 8/npm 5] <The bug can be reproduced on the https://engineering.cerner.com/terra-ui/components/terra-paginator/paginator/progressive-paginator-controlled website>
* Webpack Version: <The bug can be reproduced on the https://engineering.cerner.com/terra-ui/components/terra-paginator/paginator/progressive-paginator-controlled website>
* Operating System and version (desktop or mobile): Windows 10 desktop

**Comment by dkasper-was-taken** [comment]
This appears to be the consistent focus fallback behavior for IE11, where focus and the loss of focus are handled differently than more modern browsers. 

From offline discussion with our a11y associates this would be the expected behavior for a user familiar with IE11, and therefore shouldn't be altered from the browser default in the case of the terra-paginator. However, this does raise the larger topic of Cerner's wider approach to disabling interactive elements that the user could be actively focused upon. We've seen similar cross-browser oddities with form input fields with validation and other focus dependent interactions triggering updates.

Another issue also presented itself. It appears the combination of terra-button's virtual focus management and restoring from a previously focused, but disabled state can cause multiple focus indications. i.e. select last, tab to another numerical page, select that page, observe 2 focus rings.

We may need to discuss focus/tab-index management for disabled elements on initial disabling, as well as whether or not actions that disable UI elements should have a common, browser independent, implementation to provide a users with a Cerner standard expectation.

The other side to consider is the long term viability of IE, and Cerner's eventual, if distant, drop of support. A targeted fix may be unnecessary if more modern browsers are in alignment.

@scottwilmarth Would you care to add anything I missed from our discussion?

**Comment by KunigundaLauly** [comment]
Thanks for the response.
Should we close this ticket then?
+ Ticket for the multiple focus bug you mentioned: https://github.com/cerner/terra-core/issues/3429
